### PR TITLE
Photogram Ajax

### PR DESCRIPTION
**Pull request by kcalva**

### CODE REVIEW

File: app/views/comments/edit.js.erb
Comment (AdolfoNava) [inline app/views/comments/edit.js.erb:1]: Good on you for making these jquery statements with var

File: app/views/follow_requests/create.js.erb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+var follow_view = $("<%= j (render 'follow_requests/follow_unfollow', sender: @follow_request.sender, recipient: @follow_request.recipient) %>");
```
Comment (hcontr4) [inline app/views/follow_requests/create.js.erb:1]: Good use of a variable to hold replacement hmtl.

File: app/views/follow_requests/destroy.js.erb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+var follow_view = $("<%= j (render 'follow_requests/follow_unfollow', sender: @follow_request.sender, recipient: @follow_request.recipient) %>");
```
Comment (hcontr4) [inline app/views/follow_requests/destroy.js.erb:1]: Good use of a variable to hold replacement hmtl.

**Review by hcontr4 — APPROVED** [review]

File: app/views/comments/update.js.erb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+var updated_comment = $("<%= escape_javascript(render 'comments/comment', comment: @comment) %>");
```
Comment (AdolfoNava) [inline app/views/comments/update.js.erb:1]: you can shorten this by typing `j(render 'comments/comment', comment: @comment)` but I totally understand that you wouldn't want to shorten it.

Comment (kcalva) [reply app/views/comments/update.js.erb:1]: I did for the rest of them. I left this one so I can reference it if i ever get confused on what the shorter form does.

**Review by AdolfoNava — APPROVED** [review]
Everything looks fine

File: app/controllers/comments_controller.rb [hunk 4]
```diff
@@ -59,23 +62,27 @@ def destroy
     respond_to do |format|
       format.html { redirect_back fallback_location: root_url, notice: "Comment was successfully destroyed." }
       format.json { head :no_content }
+      format.js do
```
Comment (ahuynh3a) [inline app/controllers/comments_controller.rb:65]: great job deleting uncessary whitespaces!

File: app/views/comments/_form.html.erb [hunk 5]
```diff
@@ -1,5 +1,5 @@
-<li class="list-group-item">
-  <%= form_with(model: comment) do |form| %>
+<li id="<%= dom_id(comment.photo)%>_<%= dom_id(comment)%>_form" class="list-group-item">
+  <%= form_with(model: comment, local:false) do |form| %>
```
Comment (ahuynh3a) [inline app/views/comments/_form.html.erb:2]: great job being mindful of when to use local:false and remote:true

File: app/views/comments/create.js.erb [hunk 6]
```diff
@@ -0,0 +1,9 @@
+var added_comment = $("<%= escape_javascript(render 'comments/comment', comment: @comment) %>");
```
Comment (ahuynh3a) [inline app/views/comments/create.js.erb:1]: Good!

File: app/views/comments/edit.js.erb [hunk 7]
```diff
@@ -0,0 +1,3 @@
+var edit_form = $("<%= escape_javascript(render 'comments/form', comment: @comment) %>");
```
Comment (ahuynh3a) [inline app/views/comments/edit.js.erb:1]: I love that you created variables for your each jquery!

**Review by ahuynh3a — APPROVED** [review]
Great Job Kevin! Your code is so clean and I love that you approached your queries the long way and short way to practice!

**Pull request merged**